apps/generator/test/integration.test.js

```diff
@@ -55,4 +55,36 @@ describe('Integration testing generateFromFile() to make sure the result of the
     const file = await readFile(path.join(outputDir, testOutputFile), 'utf8');
     expect(file).toMatchSnapshot();
   });
+
+  it('should ignore specified files with noOverwriteGlobs', async () => {
+    const outputDir = generateFolderName();
+    // Manually create a file to test if it's not overwritten
+    await mkdir(outputDir, { recursive: true });
+    // Create a variable to store the file content
+    const testContent = '<script>const initialContent = "This should not change";</script>';
+    // eslint-disable-next-line sonarjs/no-duplicate-string
+    const testFilePath = path.normalize(path.resolve(outputDir, testOutputFile));
+    await writeFile(testFilePath, testContent);
+    const fileContentBefore = await readFile(testFilePath, 'utf8');
+    // Check if the file was created
+    expect(fileContentBefore).toBe(testContent);
```

Comment:
do we really need line 68-70? seems like we are testing here not generator code but just the FS write function  🤔

Reply:
I would think so? The purpose of `should ignore specified files with noOverwriteGlobs` test is to make sure that the files specified by `noOverwriteGlobs` is not overwritten. Thus we need to check the file content after the generation.

Reply:
> Thus we need to check the file content after the generation.

exactly, and this is what lines 82-84 are for, where you check if content is still there after generation

in line 68-70 you basically test `fs`, if `writeFile` was successful, which I don't think is necessary